system/ioctl.go

```diff
@@ -86,80 +96,103 @@ func getUbiDeviceSize(file *os.File) (uint64, error) {
 	return reservedSectors * sectorSize, nil
 }
 
-// Returns value in first return. Second returns error condition.
-// If the device is not a block device NotABlockDevice error and
-// value 0 will be returned.
-func ioctlRead(fd uintptr, request ioctlRequestValue) (uint64, error) {
-	var response uint64
-	_, _, errno := syscall.Syscall(syscall.SYS_IOCTL, fd,
-		uintptr(unsafe.Pointer(request)),
-		uintptr(unsafe.Pointer(&response)))
-
-	if errno == syscall.ENOTTY {
-		// This means the descriptor is not a block device.
-		// ENOTTY... weird, I know.
```

Comment:
The comment about `ENOTTY` has gotten lost and I think this is quite unobvious information. Could you make sure this is preserved somewhere?